pkg/storage/couchdb/couchdbstore.go

```diff
@@ -155,12 +155,7 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 		return errors.New("key and value are mandatory")
 	}
 
-	var valueToPut []byte
-	if isJSON(v) {
-		valueToPut = v
-	} else {
-		valueToPut = wrapTextAsCouchDBAttachment(v)
-	}
+	valueToPut := wrapTextAsCouchDBAttachment(v)
```

Comment:
we shouldn't save data in Attachment it's bad for performance. we should check if data format is json will save in fields

Reply:
you delete the correct behaviour

Reply:
in that case, implementation is incompatible since we cannot use keys such as "_key, _rev, etc."

Reply:
it looks like one implementation allows you to do that and another does not.

Reply:
@fqutishat  I have changed the logic, review it again